### ISSUE TITLE
Imperfect Test

### ISSUE THREAD

**Issue by naaodey** [bob-carpenter/diagnostic-testing#5]
Hello Professors @bob-carpenter and @andrewgelman my name is Naa Odey an MPhil student in Kwame Nkrumah University of Science and Technology (KNUST). Please I am trying to work on the gap in your paper "to estimate prevalence from imperfect tests on a non-representative sample". Please I would like to know the meaning of an imperfect in this case. To my understanding all tests are not perfect. I would will need a little clarification on that. Thank you

**Comment by mitzimorris** [comment]
hi Naa, 

you are correct that all tests are not perfect, but often the test results are used in a model as if they were 100% accurate.   This paper provides a model of the ways in which the test is not perfect:
 - a positive result when the true condition is negative
 - a negative result when the true condition is positive.

**Comment by naaodey** [comment]
Thank you
That means, the focus is on the false negative and false positive?

**Comment by bob-carpenter** [comment]
Yes, sensitivity is probability test is positive given that patient has the disease, whereas specificity is probability test is negative given that patient does not have the disease.  Together these define the bias of the test if they are not equal.  See:  https://en.wikipedia.org/wiki/Sensitivity_and_specificity

Given disease prevalence, you can use sensitivity and specificity to work out false positive, false negative, true positive, and true negative rates.  This is often the "hello world" example for statistical inference (it works in either a Bayesian or frequentist setting).

**Comment by naaodey** [comment]
Thank you all @bob-carpenter and @mitzimorris. I appreciate it
@bob-carpenter I will read the file and get back if anything. Thank you
